### PR TITLE
First pass at including all results

### PR DESCRIPTION
**Pull request by trouni**
This requires https://github.com/dmbf29/predictor-api/pull/101

@jjmountain Here is the skeleton for the results feature. It would be amazing if you could combine with what you've started with the percentages 🙏 

For #154 

image

image

### CODE REVIEW

File: src/components/LeaderboardResults.vue [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
+        m => formatDate(new Date(m.kickoffTime))
+      )
+    },
+    predictions() {
+      const predictions = {}
+      Object.values(this.matchesWithResults).forEach(dayMatches => {
+        dayMatches.forEach(match => {
+          const results = {}
+          Object.keys(this.leaderboard.results[match.id]).forEach(choice => {
+            results[choice] = this.leaderboard.results[match.id][choice].map(
```
[the remark this answers is not in the document]
Comment (trouni) [reply src/components/LeaderboardResults.vue:44]: Ah yes, the work from dmbf29/predictor-api#101 is required for this to work. You can use the [feat/ongoing-matches-predictions](https://github.com/dmbf29/predictor-api/tree/feat/ongoing-matches-predictions) branch from the API until it's approved and merged.

**Comment by jjmountain** [comment]
@trouni I'm running the rails api from `feat/ongoing-matches-predictions` and on this branch on predictor-vue but both the userId and leaderboard are not coming through the results component. In fact they are not coming through on any components.

**Comment by trouni** [comment]
@jjmountain It says "Join or create a leaderboard to get started!", I'm assuming that the user you are currently logged in as does not have any leaderboard.

**Comment by jjmountain** [comment]
The weird thing is that I have the leaderboard on main branch of predictor-api but not here. I guess it makes total sense because I hadn't joined a leaderboard on the original development database. 

The thing is I'm not able to join the lewagon league because I don't know the password for it in dev. Could you slack it to me? 🙏

**Comment by jjmountain** [comment]
@trouni Ok i figured out what was wrong (wow that one was driving me crazy..!)

I *thought* I had joined the leaderboard already but there are several called 'Le Wagon' and it turned out it was the wrong one for World Cup 2022 😆

**Review by dmbf29 — APPROVED** [review]
👍

**Pull request merged**